### DISCUSSION TITLE
Feature requests!

### DISCUSSION THREAD

**Discussion by Kvikku** [Kvikku/IntuneAssignments#4] [Ideas]
Please submit any feature requests here!

**Comment by ak47uk** [comment]
Thanks for the project, looks promising! Is it possible to bulk add/remove groups to be excluded from a policy? I have profiles assigned to the All Devices group, I would like to exclude a device group from multiple config profiles to test. 

Feedback:
Where I have manually added this device group as excluded in Intune admin, IntuneAssignments shows the policy as assigned to the excluded group with no flag to say it is excluded. 
On the Wiki under API Permissions, the group says "Groups.Read.All" but should be "Group.Read.All".

**Reply by Kvikku** [reply]
It should be possible to add "exclude" as an option for assignments. I'll add it to the list of upcoming features!

I'll also add a visual element indicate whether the group is included or excluded

Lastly I have updated the wiki to show "Group.Read.All".

Thank you for your feedback - I will get to work on these features ASAP.
